src/utils/units.hpp

```diff
@@ -0,0 +1,142 @@
+//========================================================================================
+// (C) (or copyright) 2024. Triad National Security, LLC. All rights reserved.
+//
+// This program was produced under U.S. Government contract 89233218CNA000001 for Los
+// Alamos National Laboratory (LANL), which is operated by Triad National Security, LLC
+// for the U.S. Department of Energy/National Nuclear Security Administration. All rights
+// in the program are reserved by Triad National Security, LLC, and the U.S. Department
+// of Energy/National Nuclear Security Administration. The Government is granted for
+// itself and others acting on its behalf a nonexclusive, paid-up, irrevocable worldwide
+// license in this material to reproduce, prepare derivative works, distribute copies to
+// the public, perform publicly and display publicly, and to permit others to do so.
+//========================================================================================
+#ifndef UTILS_UNITS_HPP_
+#define UTILS_UNITS_HPP_
+
+#include "artemis.hpp"
+
+#include <cmath>
+
+namespace ArtemisUtils {
+
+enum class UnitSystem { scalefree, cgs };
```

Comment:
A unit system for PPDs would be `Msolar` `AU` `yr/(2*pi)` and `K`. I think the CGS temperature unit makes it not "scalefree"? I would even call this the `ppd` unit system.

Reply:
I'm not sure we need unit systems beyond scalefree and cgs; these really only serve to have scalefree problems with trivial physical constants, or have a conversion to physical units. Whether we convert to length in cm or length in m or length in AU is already handled by the conversion between code and physical units,.

Reply:
I suppose it's just a convenience in the input file if `units = ppd` populates all the `length`, `time`, `mass`, etc. for me.

Reply:
I added a `ppd` option for `<artemis> unit_system = ` with your proposed unit conversions so this convenience should actually be available that way. :+1: